src/pal/src/init/pal.cpp

```diff
@@ -382,6 +385,20 @@ Initialize(
         }
 #endif // ENSURE_PRIMARY_STACK_SIZE
 
+        char* useDefaultBaseAddr = getenv("COMPlus_UseDefaultBaseAddr");
```

Comment:
This switch disables a OS security feature. We cannot have this config switch in the product.

Reply:
Could this be added under some `#ifdef`?

Reply:
Putting it under ifdef like `FEATURE_ENABLE_NO_ADDRESS_SPACE_RANDOMIZATION` should be ok (similar to existing `FEATURE_ENABLE_NO_RANGE_CHECKS`).

Reply:
I've put setup and usage of this parameter under `ifdef FEATURE_ENABLE_NO_ADDRESS_SPACE_RANDOMIZATION`